CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## vX.X.X (to be published)
+
+_*Breaking Changes:*_
```

Comment:
I would not consider this a breaking change...aside from the demos, we're only (effectively) unflipping the search icon in the Header bar.